spec/serializers/disability_claim_detail_serializer_spec.rb

```diff
@@ -44,4 +44,23 @@
       expect(attributes['waiver_submitted']).to eq true
     end
   end
+
+  context 'with items in vbaDocuments' do
+    let(:raw_data) do
+      fixture_file_name = "#{::Rails.root}/spec/fixtures/disability_claim/claim-with-documents.json"
+      File.open(fixture_file_name, 'rb') do |f|
+        raw_claim = f.read
+        JSON.parse raw_claim
+      end
+    end
+    let(:disability_claim) do
+      FactoryGirl.build(:disability_claim, data: raw_data)
+    end
+    let(:other_documents_count) do
+      attributes['events_timeline'].count { |obj| obj['type'] == 'other_documents_list' }
+    end
+    it 'should not duplicate documents' do
```

Comment:
Only documents without tracked item id are being pulled out of that list though
`docs = sub_objects_of('vbaDocumentList').select { |obj| obj['trackedItemId'].nil? }`
I understand what was meant by this description, but the code itself doesn't handle duplicates, it only checks the tracked item id.